docs/README.md

```diff
@@ -177,24 +177,74 @@ with a rule like [`copy_to_bin`](https://docs.aspect.build/aspect-build/bazel-li
 rules_js automatically mirrors the `bin` field from the `package.json` file of your npm dependencies
 to a Starlark API you can load from in your BUILD file or macro.
 
-For example, if you depend on the `typescript` npm package, you write this in `BUILD`:
+The format of the generated starlark file is `@<npm external repo name>//<package name where npm package is linked>:<npm package name>/package_json.bzl`.
+
+`<package name where npm package is linked>` is the name of the *bazel* package where the npm package is linked. Linking npm packages happens with `npm_link_package` or
+`npm_link_all_packages`. When [`npm_link_package`](./npm_link_package.md) is used, the `root_package` parameter specifies the bazel package. When `npm_link_all_packages` is used, the bazel package is the location where `npm_link_all_packages` was called.
```

Comment:
Ahh. The `root_package` is something different altogether. It is the package that the `node_modules/.aspect_rules_js` virtual store is located at. This is typically the bazel package where the pnpm lock file is located. So the correct description here is that for both `npm_link_package` and `npm_link_all_packages`, the "package name where npm package is linked" is the package that those targets are located in.